configs/envoy_front_proxy_v2.template.yaml

```diff
@@ -31,7 +31,8 @@
   {%endif %}
     filters:
     - name: envoy.http_connection_manager
-      config:
+      typed_config:
+        '@type': type.googleapis.com/envoy.config.filter.network.http_connection_manager.v2.HttpConnectionManager
```

Comment:
Super nit, not sure if we should always use `"` instead of `'` for this.

Reply:
Good catch. Fixing now.